### PR TITLE
Add FourCCs defined in ISO/IEC 15444-16 (J2K-in-HEIF)

### PR DESCRIPTION
**Pull request by palemieux**
See https://www.iso.org/standard/76647.html

### CODE REVIEW

File: CSV/brands.csv [hunk 1]
```diff
@@ -26,19 +26,19 @@ avc1,Advanced Video Coding extensions,ISO
 avci,AVC image and image collection brands,HEIF
 avcs,AVC image sequence brands,HEIF
 bbxm,Blinkbox Master File: H.264 video and 16-bit little-endian LPCM audio,Blinkbox
-CAEP,Canon Digital Camera,Canon
-CDes,Convergent Designs,Convergent
 caaa,CMAF Media Profile - AAC Adaptive Audio,CMAF
 caac,CMAF Media Profile - AAC Core,CMAF
+CAEP,Canon Digital Camera,Canon
```
Comment (cconcolato) [inline CSV/brands.csv]: Not sure if the chosen order was not using upper case first and then lower case. Maybe should be in a different PR?

Comment (palemieux) [reply CSV/brands.csv]: Which command is used to sort?

Comment (dwsinger) [reply CSV/brands.csv]: Also, something (Excel) thinks that DECE10 was a date and changed it to 10-Dec. Maybe re-do the PR without changing rows you don't intend to touch? (and we will work out the canonical sort separately).

**Comment by palemieux** [comment]
@dwsinger Added the proposed registrations without sorting, as requested at https://github.com/mp4ra/mp4ra.github.io/pull/41#discussion_r286263006

**Comment by dwsinger** [comment]
LGTM

**Comment by palemieux** [comment]
@dwsinger @cconcolato Any other step needed to merge?

**Comment by dwsinger** [comment]
I'm happy; if JPEG and @cconcolato  are happy, we can merge

**Comment by dwsinger** [comment]
approved at JPEG bruseels
